src/org/zaproxy/zap/extension/ascanrules/BufferOverflow.java

```diff
@@ -135,7 +148,7 @@ public void scan(HttpMessage msg, String param, String value) {
 				return;	
 
 			
-		} catch (Exception e) {
+		} catch (IOException e) {

```

Comment:
This change should be mentioned in the commit message (like it was done in the PR description).